src/content/docs/en/guides/images.mdx

```diff
@@ -240,6 +240,22 @@ By default, the `<Image />` component will produce a `.webp` file.
 - a preset (`low`, `mid`, `high`, `max`) that is automatically normalized between formats.
 - a number from `0` to `100` (interpreted differently between formats).
 
+##### inferSize
+
+<p><Since v="4.4.0" /></p>
+
+Allows you to set the original `width` and `height` of a remote image automatically.
+
+By default, this value is set to `false` and you must manually specify both dimensions for your remote image. If used with an unauthorized remote image, a request will still be made to the image domain, so the dimensions can be fetched.
```

Comment:
How is using with an unauthorized remote image different from an authorized one? Is there a situation where you would want/need to fetch the dimensions, even if you're not allowed to fetch the image?

I could see this being useful if there's a common thing you'd want to do that requires that, and we explain that you can do what that thing is. Here, I'm a left a little confused thinking, "OK, so what? So, it's the same thing."

Reply:
Good question, the basic usecase is if you want the image to be processed by sharp. Currently if a remote <Image /> is authorized it will be served by the astro site, if it is unauthorized it will become an <img src="www.example.com/png.png" />. There might be some use cases for doing this, as the Image component still adds some optimzations even to unauthrized images ( `loading=lazy`, `decoding=async`, `width`, `height` ) 

The point of authorization is SSR security, it is hypothetically possible that some image hosting domain goes rogue and starts serving malware instead of images, so authorization implies that your astro web server will make a request to the image domain and you are okay with that

inferSize makes a request to the image domain, and when it is added to an unauthorized image it will result in a <img src="www.example.com/png.png" /> 

I think it's worth documenting but I agree its difficult to convey concisely why its important 

Basically:

Authorized without inferSize = requires width / height results in your own static version of the image
Unauthorized without inferSize = requires width / height but keeps the img "dynamic" so if the domain changes the image served at that url website will change 

Authorized with inferSize = your own static version of image
Unauthorized with inferSize = "dynamic" version of img

Reply:
OK, gotcha. I propose then that we mention it, but *after* the code snippet. This isn't necessarily "up front" info to distract from just how to generally use it. So, I'd scrap that sentence from this paragraph, then after the code snippet, add something in like...

> `inferSize` can fetch the dimensions of a [remote image from a domain that has not been authorized](#authorizing-remote-images), however the image itself ...

And fill in the blank with what will/won't happen. Would something like that work?

Reply:
Sounds good I'll do that